contracts/btc_swap/target/ink/btc_swap.json

```diff
@@ -0,0 +1,969 @@
+{
```

Comment:
I think we need to add `./contracts/*/target` to `.gitignore`

Reply:
Are you sure? I intentionally added the target files here since they're used by the tests

Reply:
Hmm I see, is there anyway to build these from the tests? Right now if something breaks with ink! we won't catch it automatically in the CI since it will use the already built WASM.

Reply:
> Hmm I see, is there anyway to build these from the tests? Right now if something breaks with ink! we won't catch it automatically in the CI since it will use the already built WASM.

I'm not sure, the contracts pallet itself uses a similar approach to us. With the current approach, we check whether already compiled/deployed contracts keep working. It's probably possible but I think for now the current approach is fine. If you want to, we can add a new step to the ci to do a cargo check of the ink projects